.github/workflows/pr-garbage-collection.yml

```
@@ -1,4 +1,4 @@
-name: Clean
+name: Merge cleanup
```

I'm not sure about the names here, can we use something more specific?
And can we make the workflow and file names similar? 

```suggestion
name: Clean up Google Cloud instances
```